### PR TITLE
readme installation instructions

### PR DESCRIPTION
**Pull request by FranCogMents**
### What does this PR do?
Updates readme with local development installation instructions

### Status
READY

CC: @ethanlee16

### CODE REVIEW

**Comment by FranCogMents** [comment]
> LGTM! Wondering if we should just add the gems directly to the gemfile instead. Also not sure if we should include anything about the `config/database.yml` file that needs to be created locally.

@clairelin135 Good catch on the `config/database.yml` I'll add that note right now. I'll also put a note that `mailcatcher` should not be put into the gemfile because their documentation says not to put it in since it will conflict with the application's gems in the future. [Mailcatcher](https://mailcatcher.me/)

**Comment by clairelin135** [comment]
Interesting 🤔🤔 makes sense to not directly add mailcatcher then!

**Comment by ethanlee16** [comment]
I think we can actually remove config/database.yml from the gitignore file and directly add what you have to the repo. Not sure why it got removed in the first place.

**Comment by ethanlee16** [comment]
Maybe point out that this now requires Node 10.x as of #138?

**Comment by FranCogMents** [comment]
> I think we can actually remove config/database.yml from the gitignore file and directly add what you have to the repo. Not sure why it got removed in the first place.

perfect! I'll do that

**Comment by FranCogMents** [comment]
> Maybe point out that this now requires Node 10.x as of #138?

sounds good!

**Comment by FranCogMents** [comment]
@ethanlee16 do we have a specific postgresql version we're running?

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+# available as an environment variable. For example:
+#
+#   DATABASE_URL="postgres://myuser:mypass@localhost/somedatabase"
+#
+# You can use this database configuration with:
+#
+#   production:
+#     url: <%= ENV['DATABASE_URL'] %>
+#
+production:
```
Comment (ethanlee16) [inline config/database.yml]: One thing: can you replace this production bit with what's suggested above in the comment?

```yaml
 production:
     url: <%= ENV['DATABASE_URL'] %>
```

Comment (FranCogMents) [reply config/database.yml]: @ethanlee16 doneee 🙌🏼

Comment (ethanlee16) [reply config/database.yml]: Checking this in staging!

**Pull request merged**